pkg/runtime/runtime.go

```diff
@@ -522,7 +523,12 @@ func (m *MosnRuntime) initSecretStores(factorys ...*msecretstores.SecretStoresFa
 	// 1. register all factory methods.
 	m.secretStoresRegistry.Register(factorys...)
 	// 2. loop initializing
+	prefix := common.GetPrefixConfigFilePath()
 	for name, config := range m.runtimeConfig.SecretStoresManagement {
+		// join secretsFile path and check is it exist.
+		if config.Type == "local.file" {
+			config.Metadata["secretsFile"] = prefix + config.Metadata["secretsFile"]
```

Comment:
We shouldn't put these component-specific logic at runtime.go
You can write a new "wrapper" component under `components/secret`, which wraps the existing `local.file` secret component.
This new component is responsible for converting the path and delegating to the existing `local.file` secret component

Reply:
I will change it